manifests/params.pp

```diff
@@ -34,6 +33,34 @@
   $hiera_config        = '$confdir/hiera.yaml'
   $syslogfacility      = undef
 
+  # Windows prefixes normal paths with the Data Directory's path
+  $dir_prefix = $::osfamily ? {
+    'windows' => 'C:/ProgramData/PuppetLabs/puppet',
+    default   => '',
+  }
+
+  # Windows also leaves the 'puppet' dir off the end of the paths...
+  case $::osfamily {
+    Windows : {
+      $dir    = "${dir_prefix}/etc"
+      $logdir = "${dir_prefix}/var/log"
+      $rundir = "${dir_prefix}/var/run"
+      $ssldir = '$confdir/ssl'
+    }
+
+    default : {
+      $dir    = "${dir_prefix}/etc/puppet"
```

Comment:
You are correct. Like in my last comment, I had intended for this to accommodate alternate install paths such as D: on Windows and /usr/local on Linux. I am removing the param and will clean this up too.